packages/million/dom.ts

```diff
@@ -158,7 +147,7 @@ export const setAttribute = (
     el[name] !== undefined &&
     el[name] !== null &&
     !(el instanceof SVGElement) &&
-    setHas$.call(NON_PROPS, name)
+    NON_PROPS.has(name)
```

Comment:
We should keep the prototype cache here. Prototype cache can skip the prototype chain lookup.

Reply:
**Chrome**

<img width="1293" alt="image" src="https://user-images.githubusercontent.com/40715044/229991455-e71edfdf-84e7-4fad-8a2e-0a5a594939b5.png">

**Firefox**

<img width="1534" alt="image" src="https://user-images.githubusercontent.com/40715044/229991402-556a4843-1d68-40ed-8f2a-bbe8d84660cf.png">

Reply:
good catch!